gm4_trash_cans/data/gm4_trash_cans/functions/disable/blacklist.mcfunction

```diff
@@ -0,0 +1,24 @@
+# @s = trash_can
+# run from check_dropper
+
+# Netherite items can not be burned
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_helmet"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_chestplate"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_leggings"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_boots"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_sword"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_shovel"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_pickaxe"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_axe"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_hoe"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:ancient_debris"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_scrap"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_ingot"}] run tag @s add gm4_trash_disable
+execute if data block ~ ~ ~ Items.[{id:"minecraft:netherite_block"}] run tag @s add gm4_trash_disable
```

Comment:
```suggestion
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:netherite_chestplate"}] run tag @s add gm4_trash_disable
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:netherite_leggings"}] run tag @s add gm4_trash_disable
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:netherite_boots"}] run tag @s add gm4_trash_disable
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:netherite_sword"}] run tag @s add gm4_trash_disable
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:netherite_shovel"}] run tag @s add gm4_trash_disable
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:netherite_pickaxe"}] run tag @s add gm4_trash_disable
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:netherite_axe"}] run tag @s add gm4_trash_disable
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:netherite_hoe"}] run tag @s add gm4_trash_disable
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:ancient_debris"}] run tag @s add gm4_trash_disable
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:netherite_scrap"}] run tag @s add gm4_trash_disable
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:netherite_ingot"}] run tag @s add gm4_trash_disable
execute unless entity @s[tag=gm4_trash_disable] if data block ~ ~ ~ Items.[{id:"minecraft:netherite_block"}] run tag @s add gm4_trash_disable
```

Reply:
^ this is really important in this environment.

Reply:
You can probably minify this down to
```mcfunction
tag @s add gm4_trash_disable
execute unless data ... unless data ... unless data ... run tag @s remove gm4_trash_disable
```